### PR TITLE
changed error message to be more specific for syncing error

### PR DESCRIPTION
**Pull request by pwills11**
changed error message to be more specific for syncing error

### CODE REVIEW

**Review by ywrsusan — APPROVED** [review]
looks good

File: pybossa/view/projects.py [hunk 1]
```diff
@@ -3121,7 +3121,7 @@ def sync_project(short_name):
         current_app.logger.info(
             'exception type is {}'
             .format(str(exception_type)))
-        msg = gettext('An unexpected error occurred while trying to sync your project')
+        msg = gettext('Error: Ensure PROD API key is used, your PROD account is Subadmin & enabled, & target project is enabled for project syncing')
```
Comment (kbecker42) [inline pybossa/view/projects.py:3124]: This looks good, although I am thinking if the error message could be made more clear by changing `&` to `and`.

**Review by kbecker42 — COMMENTED** [review]
Thank you.

File: pybossa/view/projects.py (hunk 1 above)
Comment (pwills11) [reply pybossa/view/projects.py:3124]: sure, i can do this now. 
I was using the language provided by Tim. If you think any other changes are necessary, let me know!

**Review by kbecker42 — APPROVED** [review]

**Pull request merged**